cloudformation/elk-stack.template

```diff
@@ -439,6 +444,12 @@
                         "FromPort": "6379",
                         "ToPort": "6379",
                         "CidrIp": "0.0.0.0/0"
+                    },
+                    {
+                        "IpProtocol": "tcp",
+                        "FromPort": "80",
+                        "ToPort": "80",
+                        "CidrIp": "10.249.0.0/16"
```

Comment:
What is `10.249.0.0/16` exactly? may it be configurable as a parameter of the script